frontend/src/main/components/CourseDetails/CourseDetailsTable.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import OurTable from "main/components/OurTable";
+import {
+  convertToFraction,
+  formatInstructors,
+  formatLocation,
+  formatTime,
+} from "main/utils/sectionUtils.js";
+
+export default function PersonalSectionsTable({ details }) {
```

Comment:
Instead of copying `PersonalSectionsTable` with the exact same code, tests, and storybook, you can simply reuse `PersonalSectionsTable`.

In a future PR, you can consider renaming `PersonalSectionsTable` to `CourseDetailsTable` (and maybe related fixtures files) to reflect their more general purpose.

Reply:
As a note, in the future we likely will add more information to the course details page, probably not n tabular format. Using the `PersonalSectionsTable` is good enough as a placeholder; Depending on the eventual details page layout it's yet to be decided if we will rename it `CourseDetailsTable`.

Reply:
You should not be exporting default function `PersonalSectionsTable` in a file called `CourseDetailsTable`.   And as @GlassBricks said, if the code is exactly the same, why make a new component?   That's the whole purpose of components: to be reusable pieces of code that you can use to build a user interface.